core/src/main/codegen/templates/Parser.jj

```diff
@@ -8750,9 +8759,10 @@ MORE :
     < IDENTIFIER: <LETTER> (<LETTER>|<DIGIT>)* >
 }
 
-<DEFAULT, DQID, BTID, BQID> TOKEN :
+<DEFAULT, DQID, BTID, BQID, BQHID> TOKEN :
 {
-    < PARAM_IDENTIFIER: "@" (["\u0041"-"\u005a", "\u0061"-"\u007a", "\u005f"]) (["\u0041"-"\u005a", "\u0061"-"\u007a", "\u005f", "\u0030"-"\u0039"])* >
+    < AT_PARAM_IDENTIFIER: "@" (["\u0041"-"\u005a", "\u0061"-"\u007a", "\u005f"]) (["\u0041"-"\u005a", "\u0061"-"\u007a", "\u005f", "\u0030"-"\u0039"])* >
```

Comment:
Why use "@" here instead of <AT_SYMBOL> (similar to <DOLLAR>)

Reply:
Good question! fixed.